### DISCUSSION TITLE
Support for deploying single-page apps as Azure Static Web Apps

### DISCUSSION THREAD

**Discussion by vyliniem** [microsoft/aspire#2264] [Ideas]
Currently we are packaging client-apps (React, Angular SPA's) into nginx docker-images which are deployed as container-apps. Using a compute-resource to serve static content does not feel right as Azure is also providing better suited resources for this purpose, such as Static Web Apps or just plain storage behind a Frontdoor/CDN endpoint.

We would like to see support for deploying these single-page apps as Azure Static Web Apps.

**Comment by davidfowl** [comment]
There's no support for deploying to static web assets for the first version of aspire. You can see the roadmap https://github.com/dotnet/aspire/issues/1549

**Comment by kjbetz** [comment]
I agree, I'd like to see this option as well.

**Comment by KSemenenko** [comment]
I like this, becae of Blazor web assembly

**Comment by SteveHSX7** [comment]
Agreed with comment above - we just found we can't deploy our pure WebAssembly projects using Aspire because of this.

**Reply by KSemenenko** [reply]
Yeah, let’s say next year, maybe then deployment flow will work better
